compiler/luci-interpreter/src/kernels/DepthwiseConv2D.cpp

```diff
@@ -45,7 +45,11 @@ void DepthwiseConv2D::configure()
   // (4) | int8  int8   int32 int8   | quantized per channel
   // (5) | int16 int8   int64 int16  | quantized per channel 16x8
   //
-  // We only support (1) and (3) for now.
+  // We only support (1) and (3) for now, and additionally the following:
+  //     | input filter bias  output |
+  // ----+---------------------------+
+  // (5) | int16 int16  int64 int16  |
```

Comment:
nitpick: 5-th item already in the list )

Reply:
Just copied from conv2d =\